test/evergreen.yml

```diff
@@ -2589,6 +2593,22 @@ tasks:
               ./test_fops
             fi
 
+  # FIXME-WT-13951 Re-enable MacOS testing on the Evergreen waterfall - remove this task.
+  - name: fops-macos
+    tags: ["pull_request"]
```

Comment:
We probably don't want the "`pull_request`" tag here at this stage. We may want to add this to PR testing later though

Reply:
Sounds good - 4b1a6c8c6b971dafbe5279a56a662eeb1f983748